src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/Mailbox.java

```diff
@@ -17,24 +19,28 @@ public final class Mailbox extends AggregateRoot<MailboxId> {
    */
   private final Address address;
 
-  private List<Email> emails;
+  private final Map<Email, List<Label>> emails;
```

Comment:
I'm wondering whether this should be a `Set` instead of a `List` of labels. What do you think?

Reply:
We can just use a set there. I dont think the order of labels is important